### ISSUE TITLE
Fine tuning NLLB

### ISSUE THREAD

**Issue by ibtiRaj** [facebookresearch/fairseq#4870]
### Before asking:
1. search the issues.
2. search the docs.
#### My question?
I am trying to fine-tune the NLLB model on the Moroccan Arabic and English languages. I prepared my data using the "Filtering and Preparing the Data" pipeline as described in the [ReadMe](https://github.com/facebookresearch/fairseq/tree/nllb/examples/nllb/data). The result of the Prepare Data step is the following directory:
![image](https://user-images.githubusercontent.com/50990415/202407647-82368ba1-f749-4e99-a8d6-d28410b7ff98.png)

The second step now is to fine tune the model, I used this command:

**DROP=0.1
python examples/nllb/modeling/train/train_script.py cfg=nllb200_dense3.3B_finetune_on_fbseed  cfg/dataset=bilingual cfg.dataset.lang_pairs="ary_Arab-eng_Latn"  cfg.fairseq_root=/home/docker/notebooks/khadija/Translation_with_NLLB-200/fairseq     cfg.output_dir=model_fine_tuned cfg.dropout=$DROP  cfg.warmup=10     cfg.finetune_from_model=/home/docker/notebooks/khadija/Translation_with_NLLB-200/checkpoint.pt**

The bilingual.yaml file is as follows:

![image](https://user-images.githubusercontent.com/50990415/202408838-6068a247-cc41-441d-9b80-46b35b592363.png)

But I get this error when I run the command:

![image](https://user-images.githubusercontent.com/50990415/202409485-f8b06b66-d222-49e4-bdac-57ba53d0ea3a.png)

![image](https://user-images.githubusercontent.com/50990415/202409620-50f154d4-26c2-4814-a52f-7f34d4149e74.png)

Can you help me please? Is there anything I've missed?

#### What's your environment?

 - fairseq Version : 1.0.0a0
 - PyTorch Version: 1.13.0
 - OS : Linux Ubuntu 20.04.4 LTS
 - How you installed fairseq: (`pip`, source):
 - Python version: 3.8
 - CUDA/cuDNN version: Cuda 11.1, cuDNN 8

**Comment by qunash** [comment]
Hi, could you share your insights on this issue? Were you able to fine-tune the model?
